micro-ui/web/micro-ui-internals/packages/modules/core/src/components/RoleBasedEmployeeHome.js

```
@@ -157,6 +173,15 @@ export const RoleBasedEmployeeHome = ({ modules, additionalComponent }) => {
               onClick={() => history?.push(configureData?.link)}
               style={{ padding: "0px" }}
             />,
+            <Button
+            variation="teritiary"
+            label={howItWorks?.label}
+            icon={howItWorks?.icon}
+            type="button"
+            size={"medium"}
+            onClick={() => window.open(howItWorks?.link, "_blank")}
+            style={{ padding: "0px" }}
+          />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)